app/controllers/application_controller.rb

```diff
@@ -104,7 +104,7 @@ def manage_callback
     end
 
     # Then we make a request for the Clio Manaage access token.
-    response = HTTP.headers("Content-Type" => "application/x-www-form-urlencoded").post(manage_token_url(params[:code]))
+    response = HTTP.headers("Content-Type" => "application/x-www-form-urlencoded").post(manage_token_url, form: manage_token_body(params[:code]))
```

Comment:
We don't have the problem with the [Identity authenticate](https://github.com/clio/example-third-party-application/blob/main/app/controllers/application_controller.rb#L41) call since we pass the token as a form not as params!!